src/coresyf_adaptive_thresholding/coresyf_adaptive_thresholding.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "Co-ReSyF Adaptive Thresholding",
+    "type": "AdaptiveThresholding",
+    "description": "Detect ships using Constant False Alarm Rate detector",
+    "graph": false,
+    "inputs": [
+        {
+            "identifier": "Ssource",
+            "name": "source",
+            "description": "sets source to <filepath>",
+            "type": "Raster",
+            "required": true,
+            "dimensionality": 0
```

Comment:
This is deprecated. We use `collection` and it should be `true` or `false`. You can even not specify this field if it is `false`.